presto-hive/src/main/java/com/facebook/presto/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -417,6 +417,25 @@ public void renameTable(String databaseName, String tableName, String newDatabas
         throw new PrestoException(NOT_SUPPORTED, "Table rename is not yet supported by Glue service");
     }
 
+    @Override
+    public void replaceTableParameters(String databaseName, String tableName, Map<String, String> parameters)
+    {
+        Table table = getTableOrElseThrow(databaseName, tableName);
+        try {
+            TableInput tableInput = GlueInputConverter.convertTable(table);
+            tableInput.setParameters(parameters);
+            glueClient.updateTable(new UpdateTableRequest()
+                    .withDatabaseName(databaseName)
+                    .withTableInput(tableInput));
```

Comment:
Yes. Actually we don't have any unit tests for raw `ExtendedHiveMetastore`. We rely on `AbstractTestHiveClient` to test communication with different types of metastores. Before introduction of `SemiTransactionalHiveMetastore` - `AbstractTestHiveClient` was a parent class for a kind of test you would like to see. I'm not sure adding yet another `AbstractTestHiveClient` that operates on bare `ExtendedHiveMetastore` is something worth doing, since it requires additional infra in case we want to test real HiveMetastore with docker.